tests/optim/test_oss_adascale.py

```diff
@@ -37,7 +37,7 @@ def _test_basic_func(rank, world_size, tempfile_name, test_case, oss, model=None
     _dist_init(rank, world_size, tempfile_name, backend="nccl")
 
     if model is None:
-        model = Linear(2, 2, bias=False)
+        model = Linear(2, 2)
```

Comment:
you need the bias or otherwise there isn't enough params?

Reply:
yes, exactly